src/main/java/com/redhat/devtools/lsp4ij/server/ProcessStreamConnectionProvider.java

```diff
@@ -74,7 +86,13 @@ public void ensureIsAlive() throws CannotStartProcessException {
 
     protected ProcessBuilder createProcessBuilder() {
         ProcessBuilder builder = new ProcessBuilder(getCommands());
+        // Add System environment variables
         builder.environment().putAll(EnvironmentUtil.getEnvironmentMap());
```

Comment:
need to check but this might need be needed anymore, if the var environment widget already provides the system variables

Reply:
done, now the EnvironmentUtil.getEnvironmentMap() is added only if checked box is checked. For extension ls liek Qute LS, it is added.